contracts/exchange/contracts/src/interfaces/IFillAssertions.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-exchange-libs/contracts/src/LibOrder.sol";
+import "@0x/contracts-exchange-libs/contracts/src/LibFillResults.sol";
+
+
+contract IFillAssertions {
```

Comment:
I think I'd prefer removing this and `IGetContext` in favor of just inheriting `MixinExchangeCore`. Down the line the actual implementations for these functions can potentially live in a separate mixin. I'd leave it as is for now, since I think we'll be modifying the structure of these contracts a good amount soon.